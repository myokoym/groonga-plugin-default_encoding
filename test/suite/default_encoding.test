plugin_register default_encoding/default_encoding

default_encoding
