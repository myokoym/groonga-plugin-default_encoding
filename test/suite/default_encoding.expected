plugin_register default_encoding/default_encoding
[[0,0.0,0.0],true]
default_encoding
[[0,0.0,0.0],"utf8"]
